requirements.txt

```diff
@@ -36,3 +36,8 @@ six==1.12.0
 sqlparse==0.3.0
 toml==0.10.0
 urllib3==1.25.6
+PyHamcrest==1.9.0
+ipdb==0.12
+djangorestframework==3.10.3
+-e git+https://github.com/ripoul/georide-position.git@afe45e57dd851d051d9877807909e0131b2a46f1#egg=georidePosition&subdirectory=apps
```

Comment:
this file is supposed to be sorted

Reply:
add in order to try to fix continuous integration but does not make sense 👎